js/main.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+let getRandomInteger = (min, max) => {
+  if (min < max && min >= 0 && max > 0) {
+    // Источник https://learn.javascript.ru/task/random-int-min-max
+    let rand = min + Math.random() * (max + 1 - min);
+    return Math.floor(rand);
+  }
+
+  return 'Ошибка';
```

Comment:
Судя по названию функции мы ожидаем получить число, а в случае ошибки получим строку, это может сломать программу, лучше так не делать. Важно сообщать что и где пошло не так в случае ошибки, это упрощает работу с кодом и поиск ошибок.

Обычно в случаях, когда нужно сообщить об ошибке я использую `console.error`, но линтер проекта не пропустит. Ты можешь тут кидать ошибку:

```suggestion
  throw new Error('названиеФункции - что пошло не так конкретно');
```
В описании можешь текстом описать почему случилась ошибка, грубо говоря текстом описать содержание if-а